api/api.mdx

```diff
@@ -7,60 +7,202 @@ menu: 5. api
 
 In this page we will defined the form-validation api.
 
-## ValidationResult
+## Validaton Result structures
+
+### ValidationResult
+
+A field or record validation function will return a
+_ValidationResult_
+
+Properties
+
+- **type**: Validator applied (e.g. required, email, min-lenght...).
+- **succeeded**: Wether if the validation succeeded (true) or failed (false).
+- **message**: Error message. Empty if all the validations evaluated succeeded.
 
 ```typescript
 export class ValidationResult {
-  key?: string;
   type: string;
   succeeded: boolean;
-  errorMessage: string;
+  message: string;
 }
 ```
 
-## FieldValidation
+### RecordValidationResults
 
-```typescript
-interface FieldValidationFunction {
-  (value: any, vm: any, customParams: any): ValidationResultSyncAsync;
-}
+Whenever we validate all form record entries (firedRecordValidations),
```

Comment:
Replace `firedRecordValidations` by `validateRecords`.

Right now, `validateRecords` is the current name method that we are exporting as public

Reply:
It should be _ValidateRecord_ we cannot use plural we have a single record, I think we should update it on the code, do you agree?